app/assets/javascripts/home-ux.js

```diff
@@ -39,10 +39,11 @@ $(document).ready(function () {
       item: function (data, escape) {
         if (data.firstname == undefined) {
           newClientName = data.text;
+          // TODO - client that is not created on backend should be removed from frontend data
           var newClientNameArr = newClientName.split(" ");
           $("#new-firstname-input").val(newClientNameArr[0]);
           $("#new-lastname-input").val(newClientNameArr[1]);
-          $("#new-client-form-container").show("fast");
```

Comment:
This was adding inline style "display:none;". Changed to having the needed style in a class instead.